README.md

```diff
@@ -33,6 +33,9 @@ It is also required for your system to have support for POSIX threads.
 
 ### Installation
 
+Here are some simple and straight-forward install instructions for this project.
+More detailed information can be found in the file INSTALL.
+
```

Comment:
Just a typo in line 36. And in line 37 you can link to the install guidelines file.

```suggestion
Here are simpler and more straight-forward install instructions for this project.
More detailed information can be found in the [install guidelines](INSTALL).

```